guvnor-asset-mgmt/guvnor-asset-mgmt-project/src/main/filtered-resources/kjar/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>guvnor-asset-mgmt-project</artifactId>
+  <groupId>org.guvnor</groupId>
+  <version>${version.org.kie}</version>
+
+  <packaging>jar</packaging>
+
+  <name>Guvnor - Asset Management project</name>
+  <description>Guvnor - Asset management project (includes processes to manage business assets)</description>
```

Comment:
Capitals.. (I only mention it as I see you went to the trouble of fixing elsewhere)